cookbook/tools/mcp_tools.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+import sys
+from pathlib import Path
+
+from agno.agent import Agent
+from agno.tools.mcp import MCPTools
+from mcp import ClientSession, StdioServerParameters
+from mcp.client.stdio import stdio_client
+
+
+async def main(prompt: str) -> None:
+    # Initialize the MCP server
+    server_params = StdioServerParameters(
+        command="npx",
+        args=[
+            "-y",
+            "@modelcontextprotocol/server-filesystem",
+            str(Path(__file__).parent.parent),
+        ],
+    )
+    # Create a client session to connect to the MCP server
+    async with stdio_client(server_params) as (read, write):
+        async with ClientSession(read, write) as session:
```

Comment:
Could we not do all of this inside the toolkit? You pass in the server params on init, then we create the `session` inside the toolkit? We would have to do it on initialize, since it is async. You can use `_aenter` etc.